packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -8,13 +9,22 @@ import { useRecruitmentModal } from '@/components/RecruitmentModal';
 
 import { DesktopBaseNavbar, MobileBaseNavbar } from './BaseNavbar';
 
+const headroomStyles = {
+  zIndex: 1000,
+  background: 'rgba(255, 255, 255)',
```

Comment:
🤔 You should use values from our theme for `backgrond` and `boxShadow`. Also, box-shadow shouldn't appear when the navbar is in its initial position (at the top).

Reply:
I agree with Konrad and it looks better when is larger box-shadow, beacuse now it looks like simply line. In figma it's shadow style "large".

Reply:
@KonradSzwarc @wierzbka10  styling with colors like "gray.500" would not work, because its  not chakra component. I tried to connect chakra with headroom but it didn’t t work properly, so 'shadow="lg" ' does not work. 

What value of boxShadow should I use? I don't see it on figma nowhere 🤔

Reply:
@KrystianKjjk This shadow consists of two shadows:
![shadow1](https://user-images.githubusercontent.com/46423273/131727102-4c1f41ed-8b14-4f27-a62b-1cdbcdafe49d.jpg)
![shadow2](https://user-images.githubusercontent.com/46423273/131727106-f1369913-8de0-42a4-8bde-829438f6b5bc.jpg)